source/OctoVersion.Tests/WhenStringifyingAnOctoVersionInfo.cs

```diff
@@ -7,280 +8,120 @@ namespace OctoVersion.Tests
 {
     public class WhenStringifyingAnOctoVersionInfo
     {
+        static readonly OctoVersionInfo AllVersionPartsSetTo0 = new OctoVersionInfo(0, 0, 0, "", "");
+        static readonly OctoVersionInfo AllVersionPartsSetToValues = new OctoVersionInfo(1, 2, 3, "", "");
+        static readonly OctoVersionInfo VersionWithPreReleaseTag = new OctoVersionInfo(1, 2, 3, "pre", "");
+        static readonly OctoVersionInfo VersionWithPreReleaseTagAndBuildMetadata = new OctoVersionInfo(1, 2, 3, "pre", "build");
+        static readonly OctoVersionInfo VersionWithLongPreReleaseTagAndBuildMetadata = new OctoVersionInfo(2021, 1, 3, "mark-genericDocumentStore", "Branch.mark-genericDocumentStore.Sha.fb13016f3a21d7c2058fb74ab25f19e5311c6550");
+        static readonly OctoVersionInfo VersionWithLongPreReleaseTagWithSlashesAndBuildMetadata = new OctoVersionInfo(2021,1,3,"dependabot/npm_and_yarn/source/TentacleArmy.Web/ini-1.3.6","Branch.dependabot/npm_and_yarn/source/TentacleArmy.Web/ini-1.3.6.Sha.069392d9d2d37ddb6009998b92e70963badcc666");
+
+        //xunit is blurgh - IEnumerable of object[]? ick.
+        public static IEnumerable<object[]> PreReleaseTagWithDashTestCases()
+        {
+            //format is input/output/because
+            yield return new object[] { AllVersionPartsSetTo0, "", "there is no pre-release tag" };
```

Comment:
Is it worth using a tuple here? Or an explicit type (`OctoVersionInfoTestData`?) defined elsewhere in this class?

Reply:
I'd love to use a better type - it has to be `IEnumerable<object[]>`. Not pretty.

Reply:
I meant `IEnumerable<OctoVersionInfoTestData[]>`

Or it _has_ to be of type `object[]`?

Reply:
has to be an array of object 
![image](https://user-images.githubusercontent.com/373389/129136110-41dc76d1-8811-4b2d-846b-2c66fec79039.png)